### PR TITLE
Don't fail node_modules copy on missing deps

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: cli/src/utils/getdeps.ts [hunk 1]
```diff
@@ -46,7 +50,7 @@ export async function getDependencies(projectDir: string) {
     if (dependencies.has(p)) continue;
 
     if (lockHasProperty(p)) {
-      dependencies.add(p);
+      dependencies.set(p, { optional: Boolean(lockDeps[p].optional) });
```
Comment (vogre) [inline cli/src/utils/getdeps.ts]: Boolean can be written as `!!`?

Comment (bergundy) [reply cli/src/utils/getdeps.ts]: I prefer `Boolean`.

**Comment by bergundy** [comment]
@vogre exactly.

File: cli/src/utils/getdeps.ts (hunk 1 above)
Comment (arielshaqed) [reply cli/src/utils/getdeps.ts]: You could always use the `[true][+...]` operator:
```js

> [true,false][+'']
true
> [true,false][+2]
undefined
> [true,false][+'s']
undefined
> [true][+'s']
undefined
> [true][+'']
true
> [true][+false]
true
> [true][+true]
undefined
> [true][+0]
true
> [true][+-2]
undefined
>
```

**Pull request merged**